packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -54,16 +83,21 @@ export const prepareLocalchainAccountKit = (baggage, makeRecorderKit, zcf) => {
       }),
     },
     /**
-     * @param {import('@agoric/vats/src/localchain.js').LocalChainAccount} account
-     * @param {StorageNode} storageNode
+     * @param {object} initState
+     * @param {LocalChainAccount} initState.account
```

Comment:
Agree this doesn't make sense. I suggested we rename this to something closer to `OrchestrationAccountKit` here:
https://github.com/Agoric/agoric-sdk/pull/9380/files#r1610504823

Reply:
that makes some sense. Though will `OrchestrationAccountKit` be the thing for all OrchestratrionAccounts?

Another option is `LocalChainAccountHolder`, since we will be wrapping the LCA object in an Ownable holder for rights transfer. #9087

Reply:
> Though will OrchestrationAccountKit be the thing for all OrchestratrionAccounts

I'm not sure the best approach here yet, I suppose there are multiple. 

I did not end up tackling this change here -seems like it can be picked up as part of #9212